packages/nimble-components/src/tabs/index.ts

```diff
@@ -1,10 +1,11 @@
+import { observable } from '@microsoft/fast-element';
```

Comment:
Could you update the tabs stories to show more options configurations to make it easier to experiment. Maybe something similar to select, i.e. simple tabs, many tabs, wide tabs.